Documentation/contributing/testing/e2e.rst

```diff
@@ -117,6 +117,12 @@ To run all of the Kubernetes tests, run the following command from the ``test``
 
     ginkgo --focus="K8s" -noColor
 
+To run a specific test from the Kubernetes tests suite, run the following command
+from the ``test`` directory:
+
+::
+
+    ginkgo --focus="K8s.Check iptables masquerading with random-fully" -v -dryRun
```

Comment:
This is what I got trying to run it:

```
❯ ginkgo --focus="K8s.Check iptables masquerading with random-fully" -v -dryRun
level=info msg="environment variable \"K8S_VERSION\" was not set; setting to default value \"1.19\""
Running Suite: Suite-k8s-1.19
=============================
Random Seed: 1605901565
Will run 0 of 406 specs

SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
Ran 0 of 406 Specs in 0.002 seconds
SUCCESS! -- 0 Passed | 0 Failed | 0 Pending | 406 Skipped
PASS

Ginkgo ran 1 suite in 4.874204854s
Test Suite Passed
```

I assume you wanted `K8s.*Check iptables masquerading with random-fully` (with the asterisk). I think also calling out `-dryRun` won't actually run the test would be helpful to people who are just copy-pasting.

Reply:
Argh! I tried both the options (with and without "*), but I missed that it didn't select the test with the 2nd option. Thanks! Fixed.